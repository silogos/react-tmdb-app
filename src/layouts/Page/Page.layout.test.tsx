import { describe, it, expect } from "vitest";
import PageLayout from "./Page.layout";
import { render } from "@testing-library/react";

describe("Page Layout", () => {
  it("renders correctly", () => {
    const component = render(<PageLayout />);

    expect(component.container).toMatchSnapshot();
  });
});
