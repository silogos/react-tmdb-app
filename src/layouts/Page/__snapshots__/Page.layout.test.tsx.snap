// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page Layout > renders correctly 1`] = `<div />`;
